release-please-config.json

```diff
@@ -61,6 +61,9 @@
     "plugins/destination/sqlite": {
       "component": "plugins-destination-sqlite"
     },
+    "plugins/destination/sqlite-python": {
+      "component": "plugins-destination-sqlite-python"
+    },
```

Comment:
```suggestion
```

Let's not release it for now, we can keep the plugin as a reference for a developer guide we'll add to https://docs.cloudquery.io/docs/developers/creating-new-plugin